cookbooks/cdo-apps/recipes/default.rb

```diff
@@ -115,8 +115,11 @@
 
 # Production analytics utilities.
 include_recipe 'cdo-analytics' if %w[production-daemon production-console].include?(node.name)
+include_recipe 'cdo-tippecanoe' if %w[production-daemon production-console].include?(node.name)
```

Comment:
I think if we want this installed on the `daemon` for each environment, then we don’t need this line, because it’s already being included in the `daemon` node below.  Also, we probably do NOT want to run this recipe on `production-console`.

Reply:
Oh oops that was a mistake. Removed.